Cargo.toml

```
@@ -1,18 +1,27 @@
 [package]
 name = "hugr-llvm"
 version = "0.1.0"
+description = "A general and extensible crate for lowering Hugrs into LLVM IR"
```

```suggestion
description = "A general and extensible crate for lowering HUGRs into LLVM IR"
```
Maybe we should hyperlink to the hugr crate here?